archetypes/catalog/pom.xml

```diff
@@ -67,4 +67,95 @@
+                        </goals>
+                        <configuration>
+                            <directories>
+                                <directory target="${project.build.directory}">
+                                    <archives>
+                                        <archive target="cli-data/${project.version}/cli-data.zip">
+                                            <unpack-artifacts>
+                                                <unpack-artifact groupId="io.helidon.archetypes"
+                                                                 artifactId="helidon-archetype-catalog"
+                                                                 version="${project.version}"
+                                                                 type="zip"
+                                                                 target="."/>
+                                            </unpack-artifacts>
+                                            <templates>
+                                                <template source="cli-metadata.properties.mustache"
+                                                          target="metadata.properties">
+                                                    <variables>
+                                                        <variable name="buildtoolsVersion"
+                                                                  value="${build.tools.version}"/>
+                                                        <variable name="cliVersion"
+                                                                  value="${version.plugin.helidon-build-tools}"/>
+                                                    </variables>
+                                                </template>
+                                            </templates>
+                                        </archive>
+                                    </archives>
+                                    <files>
+                                        <file target="cli-data/latest">${project.version}</file>
+                                    </files>
+                                </directory>
+                            </directories>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.soebes.maven.plugins</groupId>
+                <artifactId>echo-maven-plugin</artifactId>
+                <version>0.4.0</version>
+                <executions>
+                    <execution>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>echo</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <echos>
+                        <echo> |</echo>
+                        <echo> |                      /')    </echo>
```

Comment:
Not really a comment, but what's the ascii art ?

Reply:
see above